ui/config-overrides.js

```diff
@@ -106,6 +106,25 @@ const supportDynamicPublicPathPrefix = () => (config) => {
   return config
 }
 
+const overrideProcessEnv = (value) => (config) => {
+  const plugin = config.plugins.find(
+    (plugin) => plugin.constructor.name === 'DefinePlugin'
+  )
+  const processEnv = plugin.definitions['process.env'] || {}
+
+  plugin.definitions['process.env'] = {
+    ...processEnv,
+    ...value,
+  }
+
+  return config
+}
+
+const getInternalVersion = () => {
+  const out = execSync(`grep -v '^\#' ../release-version`)
```

Comment:
Maybe just reading the file information and process it by code is better!

Reply:
Can you explain the reasoning about this? Then I can also make better decisions next time. : D

Reply:
My suggestion is based on that it is more efficient while also code is few.

Reply:
#WIndows_Users_Lives_Matter